lib/package-detail-view.js

```diff
@@ -372,7 +372,16 @@ export default class PackageDetailView {
       readme = fs.readFileSync(this.readmePath, {encoding: 'utf8'})
     }
 
-    const readmeView = new PackageReadmeView(readme)
+    let readmeSrcBase
```

Comment:
What do you think about renaming this to `readmeSrc` instead?

Reply:
If you think that's better than I'm happy to change it. Would you want it changed in `package-readme-view.js` as well?

Reply:
Done in latest commit